File: src/main/java/com/ByTech/ByTech/Payment/repositories/PaymentRepository.java
package com.ByTech.ByTech.Payment.repositories;

import com.ByTech.ByTech.Payment.models.Payment;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.List;

public interface PaymentRepository extends JpaRepository<Payment, Long> {
    List<Payment> findAllByOrderByCreatedAtDesc();
    List<Payment> findAllByEnterpriseIdOrderByCreatedAtDesc(Long enterpriseId);
}

File: Movil/backend/ByTech_movil/src/main/kotlin/com/example/ByTech_movil/Payment/repositories/PaymentRepository.kt
package com.example.ByTech_movil.Payment.repositories

import com.example.ByTech_movil.Payment.models.Payment
import org.springframework.data.jpa.repository.JpaRepository

interface PaymentRepository : JpaRepository<Payment, Long> {
    fun findAllByOrderByCreatedAtDesc(): List<Payment>
}
